doc/releases/changelog-0.10.0.md

```diff
@@ -42,244 +46,229 @@
         return qnode()
     ```
 
-  * Specifying multiple compilation pass plugins or dialect plugins directly in :func:`~.qjit` via 
-    the `pass_plugins` and `dialect_plugins` keyword arguments, which must be lists of plugin paths.
+  - Alternatively, one or more dialect and pass plugins can be specified in advance in the
+    :func:`~.qjit` decorator, via the `pass_plugins` and `dialect_plugins` keyword arguments. The
+    :func:`~.passes.apply_pass` function can then be used without specifying the plugin.
 
     ```python
     from pathlib import Path
 
     plugin = Path("shared_object_file.so")
 
+    @catalyst.passes.apply_pass("pass_name")
     @qml.qnode(qml.device("lightning.qubit", wires=0))
     def qnode():
       qml.Hadamard(wires=0)
       return qml.state()
 
     @qml.qjit(pass_plugins=[plugin], dialect_plugins=[plugin])
     def module():
-      return catalyst.passes.apply_pass(qnode, "pass_name")()
+      return qnode()
     ```
 
-  For more information on usage, 
-  visit our [compiler plugin documentation](https://docs.pennylane.ai/projects/catalyst/en/stable/dev/plugins.html).
+  For more information on usage, visit our
+  [compiler plugin documentation](https://docs.pennylane.ai/projects/catalyst/en/stable/dev/plugins.html).
 
 <h3>Improvements 🛠</h3>
 
-* The lightning runtime now supports finite shots with measuring expectation values of `qml.Hermitian`.
-  [(#451)](https://github.com/PennyLaneAI/catalyst/pull/451)
-
-* Pybind11 has been replaced with nanobind for C++/Python bindings in the frontend and in the runtime.
-  [(#1173)](https://github.com/PennyLaneAI/catalyst/pull/1173)
-  [(#1293)](https://github.com/PennyLaneAI/catalyst/pull/1293)
-  [(#1391)](https://github.com/PennyLaneAI/catalyst/pull/1391)
-
-  Nanobind has been developed as a natural successor to the pybind11 library and offers a number of
-  [advantages](https://nanobind.readthedocs.io/en/latest/why.html#major-additions) like its ability 
-  to target Python's [stable ABI interface](https://docs.python.org/3/c-api/stable.html) starting 
-  with Python 3.12.
-
-* Catalyst now uses the new compiler API (`catalyst-cli`) to compile quantum code from the Python 
-  frontend instead of using pybind11 as an interface between the compiler and the frontend. 
+* The Catalyst CLI introduced in `v0.9` without being included in binary distributions (wheels)
```

Comment:
```suggestion
* The Catalyst command-line interface (CLI) introduced in `v0.9` without being included in binary distributions (wheels)
```

Optional

Reply:
Well the idea was that "Catalyst CLI" is the name of the tool, although I'm happy to include the expanded form in a different place in the text.